.circleci/config.yml

```diff
@@ -68,8 +68,22 @@ jobs:
       - run: *run_yarn_install
       - save_cache: *save_yarn_cache
       - run:
-          name: E2E Tests
-          command: yarn test:e2e
+          name: Integration Tests
+          command: yarn test:integration
+
+  test_e2e:
+    <<: *defaults
+    steps:
+      - checkout
+      - run: *install_yarn_version
+      - restore_cache: *restore_yarn_cache
+      - run: *run_yarn_install
+      - save_cache: *save_yarn_cache
+      - run:
+          name: End-2-End tests
+          command: |
+            yarn run website:build
+            yarn run test:e2e:saucelabs
```

Comment:
Looks like this builds the lib again, could use the ci workflow more efficiently

Reply:
I had the same issue on `instantsearch.js`, but it seems that in the current configuration each CircleCI job is isolated in it's own environment, which forces us to re-do a lot of stuff in each job (checkout, yarn install, yarn build).

IMHO it is out for scope for this PR (unless I made an error or I missed an obvious optimization :wink: ) but there is room for improvement for sure. I think we could tackle this in a later task.

Reply:
IMHO it's simpler if it's redoing it, especially if you want to rerun a particular job like it's often the case with E2E. So this is ok to me.